src/Engine/Component/Component.hpp

```diff
@@ -10,18 +10,12 @@
 
 #include <Core/Types.hpp>
 
-namespace Ra {
-
-namespace Engine {
+namespace Ra::Engine {
```

Comment:
This does not seems to comply with our code style policy.
@dlyr do you have a preference on this ?

Reply:
Yep, I agree. But it seems to me that it might be less error-prone to explicitly define the full-qualified namespace than writing nested namespace.

Reply:
Hum ok. But if we agree on this code we need to update the whole source code.

Reply:
Sure, I just do that on engine for now, but the entire code will be updated.